File: Beecrowd/1132 Múltiplos de 13/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws Exception {
        Scanner sc = new Scanner(System.in);

        int x, y, aux, soma = 0;

        x = sc.nextInt();
        y = sc.nextInt();

        if (x > y) {
            aux = x;
            x = y;
            y = aux;
        }

        for (int i = x; i <= y; i++) {
            if (i % 13 != 0) {
                soma += i;
            }
        }

        System.out.println(soma);

        sc.close();
    }
}

File: Beecrowd/1132 Múltiplos de 13/main.cpp
#include <iostream>

using namespace std;

int main()
{

    int X, Y, aux, i, soma = 0;

    cin >> X >> Y;

    if (X > Y)
    {
        aux = X;
        X = Y;
        Y = aux;
    }

    for (i = X; i <= Y; i++)
    {
        if (i % 13 != 0)
        {
            soma += i;
        }
    }

    cout << soma << endl;

    return 0;
}